README.md

```diff
@@ -90,14 +97,27 @@ circuit_handler do |handler|
  handler.invocation_timeout = 10
  handler.excluded_exceptions += [NotConsideredFailureException]
 end
+
+# An example of a make request
+multi = Px::Service::Client::Multiplexer.new
```

Comment:
This example seems wrong.  You need to include the breaker into the requesting class, at least.